### PR TITLE
Updating eventing workflows for GA

### PR DESCRIPTION
**Pull request by abrennan89**
Changes (TBC)
- Renamed `knative_eventing` dir -> `event_workflows` and updated all instances across docs (need links updates)
- Moved release notes and support to bottom of nav; no redirect required since no dir change.
- Moved triggers into brokers and triggers assembly; needs link update

### CODE REVIEW

File: serverless/event_workflows/serverless-create-kn-trigger.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[id="serverless-create-kn-trigger_{context}"]
+= Creating a trigger using `kn`
+
+All events that are sent to a broker will be sent to all of the subscribers of that broker by default.
```
Comment (lberk) [inline serverless/event_workflows/serverless-create-kn-trigger.adoc:4]: Could we maybe clarify this sentence a bit? while technically true (the best kind of true), it gave me the impression we were trying to say you use subscriptions (the channel related resource type) to specify sinks being attached to brokers and they're all enabled by default.  I think we're trying to say is: unless a filter is specified on the trigger spec, the trigger will forward all events from the broker to that sink, right?

Perhaps (and feel free to bikeshed): "By default, a Trigger will forward all events sent to a broker to it's specified subscribers." ?

Comment (abrennan89) [reply serverless/event_workflows/serverless-create-kn-trigger.adoc:4]: Makes sense to me, thanks @lberk ! I've made some modifications in line with this so lmk if that looks better to you.

**Comment by mgencur** [comment]
hi @abrennan89 , I think it looks good. 
QE Approved.

File: modules/serverless-creating-subscriptions.adoc [hunk 2]
```diff
@@ -1,29 +1,21 @@
 // Module included in the following assemblies:
 //
 // <List assemblies here, each on a new line>
-// * serverless/knative_eventing/serverless-subscriptions.adoc
+// * serverless/event_workflows/serverless-channels.adoc
 
 [id="serverless-creating-subscriptions_{context}"]
 = Creating a subscription
 
 You can create a subscription to connect a service or other event sink to a channel.
```
Comment (sheriff-rh) [inline modules/serverless-creating-subscriptions.adoc:9]: this line mirrors line 13

**Review by sheriff-rh — APPROVED** [review]
Found one line that appears to be a duplicate, but other than that - LGTM! 🌮 🐱 🥇 🎉

**Pull request merged**